addons/respawn/stringtable.xml

```diff
@@ -222,6 +222,7 @@
         </Key>
         <Key ID="STR_ACE_Respawn_FriendlyFire_Module_Description">
             <English>Using this module in your mission will make it so any friendly fire kills will be displayed in form of a message in chat.</English>
+            <German>Dieses Modul zeigt jeden Gefallenen durch Verbündetenbeschuss in Form einer Textnachricht im Chat an.</German>
```

Comment:
Freundbeschuss oder Verbündetenbeschuss?

`...beschuss, in Form einer Textnachricht, im Chat an.`